packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeMenu/search-input.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Icon, Input } from '@deriv/components';
+import { localize } from '@deriv/translations';
+
+type TSearchInput = {
+    onChange: React.ChangeEventHandler<HTMLInputElement | HTMLTextAreaElement | null>;
+    onClickClearInput: () => void;
+    value: string;
```

Comment:
Just asking, ignore if not relevant: is `value` optional here? I mean on line 17 there is a ternary operator

Reply:
Can be. No issue either way though. just left it as required because of line 21